inspirehep/dojson/conferences/fields/bd1xx.py

```diff
@@ -29,13 +29,13 @@
 from dojson import utils
 
 from ..model import conferences
+from ...utils import force_single_element
 from ...utils.geo import parse_conference_address
 
 from inspirehep.utils.helpers import force_force_list
 
 
 @conferences.over('acronym', '^111')
-@utils.for_each_value
 def acronym(self, key, value):
     """Conference acronym."""
     self['date'] = value.get('d')
```

Comment:
Uh, right, I was leaning on the previous tests here but they don't look very reliable. I think I will retract the first two commits.